README.md

```diff
@@ -2,6 +2,62 @@
 Docker image for universal postgres backups
 
 # Roadmap
-- [ ] Add support for S3
+- [X] Add support for S3
+- [X] Add CI/CD to publish image to DockerHub
 - [ ] Add retention policy settings by env vars
-- [ ] Notify about backup status by HTTP-request
\ No newline at end of file
+- [ ] Notify about backup status by HTTP-request
+- [ ] Add docker-compose example
+
+## Docker build
+```shell
+docker build . -t pg-backups:0.0.1
```

Comment:
Let's make tag name same as this repository name

Reply:
Done